godtools/App/Share/Data/GodToolsParserLogger/GodToolsParserLogger.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  GodToolsParserLogger.swift
+//  godtools
+//
+//  Created by Levi Eggert on 8/29/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+import GodToolsToolParser
+
+class GodToolsParserLogger {
+    
+    static let shared: GodToolsParserLogger = GodToolsParserLogger()
+    
+    private var isStarted: Bool = false
+    
+    private init() {
+        
+    }
+    
+    func start(firebaseCrashReporting: FirebaseCrashReporting) {
+        
+        guard !isStarted else {
+            return
+        }
+        
+        isStarted = true
+                
+        NapierProxyKt.enableCustomLogging { (logLevel: NapierLogLevel, tag: String?, throwable: KotlinThrowable?, message: String?) in
+            
+            guard let error = throwable?.asError() else {
+                return
+            }
+            
+            firebaseCrashReporting.recordError(error: error)
+        }
```

Comment:
this is logging just any captured exceptions pushed through the log system. This is not currently including any log messages that may be included with the errors.

you will want to use `firebaseCrashReporting.log()` to record any log messages that are included as well.

see: https://github.com/AAkira/Napier/blob/master/ios/Napier/AppDelegate.swift#L27

Reply:
instead of separating the logging callback into 2 separate callbacks like they did in their sample, I left it as a single callback that you can log messages and record errors from

Reply:
Thanks Daniel!  Going to update.